index.html

```diff
@@ -45,7 +46,7 @@ <h2>About:Girma Ebssa</h2>
         a diverse range of clients.
       </p>
     </section>
-    <h2>Work Experience</h2>
+    <h3>Work Experience</h3>
```

Comment:
The only missing requirement now is this should be an `<h2>` instead of an `<h3>` tag to meet all the requirements. I'd just fix that, merge this, and move on to the next class.